dagql/dagui/spans.go

```diff
@@ -43,23 +47,22 @@ type Span struct {
 // Snapshot returns a snapshot of the span's current state, incrementing its
 // Version with every call.
```

Comment:
comment out of date about Version; I was also a little surprised that calling Snapshot results in us assuming it's Remote and still not totally following why

Reply:
I *think* this makes sense - I think it's just the naming (maybe a better to think of it might be `Final`)? Once we snapshot a span, it's in its final state, there's no need to calculate anything else on it.

Reply:
Feel free to rename to your hearts content! Was definitely a hasty choice

Reply:
Excellent :D will rename, and merge :tada:

(now go enjoy PTO! :clinking_glasses: :clinking_glasses:)